### ISSUE TITLE
[BUG] Grid inconsistencies?

### ISSUE THREAD

**Issue by ether42** [revarbat/BOSL2#78]
I was trying to get some shapes in a grid but it seems the children of `grid2d` are slightly rotated.

This simple example should clearly show how children aren't lined up together:
```scad
include <BOSL2/std.scad>
grid2d(spacing=2, cols=5, rows=5) cube(2);
```

![grid_cube](https://user-images.githubusercontent.com/3048933/61405977-3e6ac680-a8db-11e9-88c4-d6223d1534f3.png)

Tested with OpenSCAD 2019.05.

**Comment by revarbat** [comment]
Verified that the cubes are slightly rotated.

**Comment by revarbat** [comment]
This is a leftover of the transition from `orient` as rotation angles to being a directional vector.
Fixed in 6ce93f75e009a4e26344a8f78f0323ff2817e8fc
